commands/mentor.py

```diff
@@ -0,0 +1,28 @@
+import requests
+import random
+from disco.bot import Bot,Plugin
+from disco.types.message import MessageEmbed
+from config_mock import AnnounceBotConfig, TestPluginConfig
```

Comment:
i don't see anything called config_mock anywhere, is this a local file not commited, seperate from config.py?

Reply:
my bad, i actually forgot to switch that over to config.py, when i was moving everything around, thanks!